modules/creating-customer-seccomp-profile.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * security/seccomp-profiles.adoc
+
+:_content-type: PROCEDURE
+[id="spo-creating-profiles_{context}"]
+= Creating seccomp profiles
+You can use the `SeccompProfile` object to create profiles.
```

Comment:
SeccompProfile will not be available when SPO not installed. Cannot be used.

Reply:
Ah that's right, thank you!